src/KafkaFlow/Compressor/IMessageDecompressor.cs

```diff
@@ -0,0 +1,15 @@
+namespace KafkaFlow.Compressor
+{
+    /// <summary>
+    /// Used to create a message decompressor
+    /// </summary>
+    public interface IMessageDecompressor
```

Comment:
Shouldn't this new split interfaces be included in the KafkaFlow.Abstractions package @filipeesch ? I Imagine that a user may want to create their own implementation of the (de)compressors and (de)serializers

Reply:
Since the Serializer is a middleware, I really don't know if it should be moved to the Abstractions package. Today we have the `KafkaFlow.Serializer` package, which works as the serialization "abstrations". For now, I would maintain in the Serializer package

Reply:
Alright

Reply:
Forget what I said. You are right @simaoribeiro . We can move these interfaces to Abstractions, since we will remove the Serializer and Compressor projects. The same applies to the `IMessaheHandler` of TypedHandler

Reply:
@jose-sousa-8 Can you make the change, please?

Reply:
Done here https://github.com/Farfetch/kafkaflow/pull/435/commits/b203a28b553cdb569330f3aff92e576d883608e7#diff-94ec3a0877627917cf892c09797d46d2a49b60d7babe0fe50671fd9ef0cea183